my-app/src/public-components/footer/footer.js

```diff
@@ -31,37 +32,38 @@ export class Footer extends Component {
   // Hide or show the footer.
   handleScroll = () => {
     
-    const { prevScrollpos } = this.state;
-  
     const currentScrollPos = window.pageYOffset;
-    const visible = prevScrollpos > currentScrollPos;
-  
+    const scrollInBottom = document.documentElement.scrollTop === document.documentElement.scrollHeight-document.documentElement.clientHeight;
+    const visible = !scrollInBottom;
+ 
     this.setState({
       prevScrollpos: currentScrollPos,
       visible
     });
+   
   };
 
   render() {
   return (
+    <div className="footer_main">
     <div className={classnames("footer", {
       "footer_hidden": this.state.visible })}>
         <div className="footer_logo_social">
           <div className="footer_logo">
-          <div>
+          <div className="footer_logo_text">
             {new Date().getFullYear()} &#9400;: 
-            <Textlink className="footer_logo_text" name=' nadiya.com' path='https://www.nadiya.com' />
+            <Textlink className="footer_logo_text" name='www.nadiya.com' path='https://www.nadiya.com' />
```

Comment:
this link will not work. react-router can redirect user only inside SPA. You provide outer link. Change it please.